### ISSUE TITLE
Stoa cannot be restarted

### ISSUE THREAD

**Issue by Geod24** [bosagora/stoa#231]
Attempting to restart Stoa on the live system yields:
```
[Stoa] 2020-12-29T03:04:15.876Z info Agora endpoint: http://127.1.1.1:4567
[Stoa] 2020-12-29T03:04:15.877Z info sqlite3 database filename: /Users/geod24/projects/bpfk/stoa/data/main.db
[Stoa] 2020-12-29T03:04:15.947Z error Failed to create LedgerStorage: Error: SQLITE_ERROR: trigger tx_trigger already exists
```

This is blocking us from making any progress and should be fixed ASAP.

**Comment by Geod24** [comment]
CC @MichaelKim20 @TrustHenry when will you have time to tackle this ?

**Comment by TrustHenry** [comment]
I'll fix it.

**Comment by Geod24** [comment]
Fixed in #233 although a test is still missing.

**Comment by TrustHenry** [comment]
I will add a test case.
